app/controllers/lead_providers/your_schools_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module LeadProviders
+  class YourSchoolsController < ::LeadProviders::BaseController
+    before_action :set_lead_provider
+
+    def index
+      @cohorts ||= Cohort.where(start_year: [2021, 2022])
```

Comment:
I think this should show all cohorts for the leadprovider, rather than a hardcoded list. LeadProvider has a `has_and_belongs_to_many :cohorts` through `cohort_lead_providers`

Reply:
I think you are right - when I first pulled the code from develop - that model was not there - Good point

Reply:
Done